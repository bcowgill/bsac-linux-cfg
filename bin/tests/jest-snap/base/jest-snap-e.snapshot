// jest-snap-e.snapshot
// in/SAMPLE.test.js.snap
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Webpack Integration Tests removes-duplicate-css 1`] = `"a{color:#00f}body{color:red}p{color:green}"`;
