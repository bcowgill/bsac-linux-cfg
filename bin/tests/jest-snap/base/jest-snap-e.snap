// jest-snap-e.snap
// in/SAMPLE.test.js.snap
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Webpack Integration Tests removes-duplicate-css 1`] = `"a{color:#00f}body{color:red}p{color:green}"`;
